backend/README.md

```diff
@@ -33,17 +33,16 @@ USING DOCKER
   - backend/target
 
 Formatting:
-- Check code formatting: `mvn formatter:validate`
-- Format the code: `mvn formatter:format`
+- Check code formatting: `mvn spotless:check`
+- Format the code: `mvn spotless:apply`
```

Comment:
document the formatting tool for frontend

Reply:
Already done